be/src/exprs/vectorized/runtime_filter_bank.cpp

```diff
@@ -369,6 +416,59 @@ void RuntimeFilterProbeCollector::update_selectivity(vectorized::Chunk* chunk) {
     }
 }
 
+// reentrant version of update_selectivity, used by pipeline engine.
+void RuntimeFilterProbeCollector::update_selectivity(vectorized::Chunk* chunk,
```

Comment:
Why write the same code again?

Reply:
its reentrant version of the non-reentrant RuntimeFilterProbeCollector::update_selectivity